src/test/java/github/weichware10/util/db/TrialsTest.java

```diff
@@ -0,0 +1,309 @@
+package github.weichware10.util.db;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import github.weichware10.util.Logger;
+import github.weichware10.util.Enums.ToolType;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Enums.ToolType' import. Should be before 'github.weichware10.util.Logger'.